public/app/dashboard/splashController.js

```diff
@@ -27,15 +27,43 @@
 
   'use strict';
 
-  angular.module('collabosphere').controller('SplashController', function(deepLinkId, me, userFactory, $scope) {
+  angular.module('collabosphere').controller('SplashController', function(deepLinkId, me, dashboardFactory, userFactory, $scope) {
```

Comment:
@paulkerschen we prefer alphabetized function args, yes? I.e., `dashboardFactory` first.